File: QE-SELENIUM-PYTHON/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/login-form/")
print("Page Title:", driver.title)
username = driver.find_element(By.ID, "username")
username.send_keys("admin")
password = driver.find_element(By.ID, "password")
password.send_keys("password")
login_button = driver.find_element(By.XPATH, "//button[text()='Submit']")
login_button.click()
driver.quit()

File: QE-SELENIUM-JAVA/demo/src/main/java/demos/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {

    public static void main(String[] args) {

        FirefoxDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/login-form/");
        System.out.println("Page Title: " + driver.getTitle());
        WebElement username = driver.findElement(By.id("username"));
        username.sendKeys("admin");
        WebElement password = driver.findElement(By.id("password"));
        password.sendKeys("password");
        WebElement loginButton = driver.findElement(By.xpath("//button[text()='Submit']"));
        loginButton.click();
        driver.quit();
    }
}